src/main/resources/application-local.yml

```diff
@@ -15,6 +15,9 @@ aws:
       attributes:
         max-messages: 1
         wait-time: 5
+    case-migrator:
+      url: ${localstack.base-url}/queue/migration-queue
```

Comment:
It might be beneficial if we're doing it like this to pull `account` on L12 out to the `aws.sqs` level, same for the `attributes`.